src/wazuh_db/wdb_parser.c

```diff
@@ -5173,6 +5182,25 @@ int wdb_parse_global_insert_agent_group(wdb_t * wdb, char * input, char * output
     return OS_SUCCESS;
 }
 
+int wdb_parse_global_select_group_belong(wdb_t *wdb, char *input, char *output) {
+    int agent_id = atoi(input);
+    char *out = NULL;
```

Comment:
We can move this declaration directly to line 5196.

Reply:
Done !